### DISCUSSION TITLE
ModelOMatic model selection still supported but not documented?

### DISCUSSION THREAD

**Discussion by lguy** [iqtree/iqtree2#300] [Q&A] [answered]
Hi there,
I've stumbled upon some code running iqtree2 (v. 2.2.5) where the authors use the --modelomatic option. I was not aware that it existed, and searching the website didn't return any result, except on the front page (in the 'Highlights of v2' box). However, it doesn't seem to be documented anywhere else, neither in the Model selection part of the documentation nor in the Command Line reference section. I could find it available on an iqtree2 version (v. 2.3.2) installed on my computer. 

Is there a reason that it is not documented anymore? Is it being deprecated? Is it OK to use it (I know, my responsibility to know, but thought I'd ask anyway)?

```
$ iqtree2 --version
IQ-TREE multicore version 2.3.2 COVID-edition for Linux 64-bit built Apr  9 2024
$ iqtree2 --help | grep modelomatic
  --modelomatic        Find best codon/protein/DNA models (Whelan et al. 2015)
```

**Comment by roblanf** [comment, marked as answer]
Hi @lguy, 

I don't know the actual reason this isn't documented anymore, but I'm going to hazard a guess that this is deprecated. 

Either way, it looks like it should be deprecated. ModelOMatic is ~10 years old, hasn't been updated in 8 years, and has a very very limited set of models that it compares (https://github.com/simonwhelan/modelomatic?tab=readme-ov-file) relative to those available in IQ-TREE. I suggest using modelfinder (e.g. `-m MF` or `-m MFP`) to select models.

I'll let @bqminh answer with the true history of this command (he may be slow, he's teaching a big class at the moment), but my guess is that this command was included for some specific purpose at some point (e.g. to compare modelomatic to other model selection algorithms).

Rob

**Comment by lguy** [comment]
Hi @roblanf,

Thanks for the explanation. I don't really need a full history of the command, knowing that it's old and should be deprecated is good enough for me. 

If anything, this highlights the value of maintaining bioinformatics resources over long periods of time, something you, @bqminh and the IQ-TREE team is excellent at, and deserve a lot of kudos for! 

Cheers,

Lionel
